internal/mode/static/nginx/runtime/manager.go

```diff
@@ -46,16 +52,23 @@ type MetricsCollector interface {
 type ManagerImpl struct {
 	verifyClient     *verifyClient
 	metricsCollector MetricsCollector
+	ngxPlusClient    *ngxclient.NginxClient
 }
 
 // NewManagerImpl creates a new ManagerImpl.
-func NewManagerImpl(collector MetricsCollector) *ManagerImpl {
+func NewManagerImpl(ngxPlusClient *ngxclient.NginxClient, collector MetricsCollector) *ManagerImpl {
```

Comment:
would it make sense to start testing Manager? using ginkgo, similar to other components

Reply:
I think we would need to build a full wrapper around the nginx client so that we can mock out its methods. It currently has no mocks or fakes that I'm aware of.

Would also need to mock the existing "verifyClient", which probably means building it into an interface of its own.

Reply:
Think that's worth a followup story, "good-first-issue"?